.changeset/moody-chicken-do.md

```diff
@@ -0,0 +1,16 @@
+---
+'@commercetools-frontend/application-components': minor
+---
+
+Add new `<FormMainPageLayout>` component.
```

Comment:
Please follow the naming conventions and remove the `Layout` part from the name (and rename the files).

```suggestion
Add new `<FormMainPage>` component.
```